apps/test/unit/code-studio/pd/workshop_dashboard/components/local_summer_workshop_survey_resultsTest.js

```diff
@@ -0,0 +1,117 @@
+import LocalSummerWorkshopSurveyResults from '@cdo/apps/code-studio/pd/workshop_dashboard/local_summer_workshop_survey_results';
+import {shallow, mount} from 'enzyme';
+import {expect} from 'chai';
+import sinon from 'sinon';
+import React from 'react';
+
+describe("Local Summer Workshop Management", () => {
+  const getFakeSurveyDataForFacilitator = () => {
+    return {
+      facilitator_breakdown: false,
+      this_workshop: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2'],
+        'num_enrollments': '10',
+        'num_surveys': '8',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5'
+      },
+      all_my_local_workshops: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2', 'Feedback 3', 'Feedback 4'],
+        'num_enrollments': '200',
+        'num_surveys': '180',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5'
+      }
+    };
+  };
+
+  const getFakeSurveyDataForWorkshopOrganizer = () => {
+    return {
+      facilitator_breakdown: true,
+      facilitator_names: ['Han', 'Chewbacca'],
+      this_workshop: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2'],
+        'num_enrollments': '10',
+        'num_surveys': '8',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5',
+        'how_clearly_presented': {'Han': 5, 'Chewbacca': 1},
+        'things_facilitator_did_well': {'Han': ['Shot first'], 'Chewbacca': ['Roared a lot']},
+        'facilitator_names': ['Han', 'Chewbacca']
+      },
+      all_my_local_workshops: {
+        'venue_feedback': ['Feedback 1', 'Feedback 2', 'Feedback 3', 'Feedback 4'],
+        'num_enrollments': '10',
+        'num_surveys': '8',
+        'received_clear_communication': '5.5',
+        'how_much_learned': '4.5',
+        'how_clearly_presented': 5
+      }
+    };
+  };
+
+  describe("View summarization", () => {
+    it("Spinner displayed initially", () => {
+      let localSummerWorkshopSurveyResults = shallow(
+        <LocalSummerWorkshopSurveyResults
+          params={{workshopId: '1', facilitators: []}}
+        />
+      );
+
+      expect(localSummerWorkshopSurveyResults.state('loading')).to.be.true;
+      expect(localSummerWorkshopSurveyResults.find('Spinner')).to.have.length(1);
+    });
+
+    it("Table gets displayed after loading is completed", () => {
```

Comment:
nit: similar to above. Suggest `"Displays table after loading is completed"`. Same with other test cases below.